### ISSUE TITLE
Error handling when printer is out of paper

### ISSUE THREAD

**Issue by siddharth5314** [DantSu/ESCPOS-ThermalPrinter-Android#163]
Greetings, does the escpos library support the scenario when the printer is out of paper ?

**Comment by DantSu** [comment]
No, a command exist but need captor on printer. That need compatible printer. My printer isn't compatible.
